.github/workflows/eval-gate.yml

```
@@ -0,0 +1,40 @@
+
+      - name: Install Dependencies
+        working-directory: servers/exarchos-mcp
+        run: npm ci
+
+      - name: Build
+        working-directory: servers/exarchos-mcp
+        run: npm run build
+
+      - name: Run Regression Evals
+        working-directory: servers/exarchos-mcp
+        run: echo '{"ci": true}' | node dist/cli.js eval-run
```

actionlint (1.7.11)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)